docs/en/stack/ml/df-analytics/hyperparameters.asciidoc

```diff
@@ -20,9 +20,22 @@ a model, and calculate its performance on validation data.
 
 You can view the hyperparameter values that were ultimately chosen by expanding
 the job details in {kib} or by using the
-{ref}/get-dfanalytics-stats.html[get {dfanalytics-job} stats API]. You can also
-see the specific type of validation loss (such as mean squared error or binomial
-cross entropy) that was used to compare each round of optimization. 
+{ref}/get-trained-models.html[get trained models API]. You can also see the 
+specific type of validation loss (such as mean squared error or binomial cross 
+entropy) that was used to compare each round of optimization using the 
+{ref}/get-dfanalytics-stats.html[get {dfanalytics-job} stats API].
+
+
+[[hyperparameter-importance]]
+== Calculating the importance of hyperparameters
```

Comment:
> == Calculating the importance of hyperparameters

Do we need to link to this specific content from elsewhere? If not, I think this paragraph can be interwoven with the existing ones.